compiler/rustc_expand/src/expand.rs

```diff
@@ -721,7 +721,11 @@ impl<'a, 'b> MacroExpander<'a, 'b> {
                 SyntaxExtensionKind::LegacyBang(expander) => {
                     let prev = self.cx.current_expansion.prior_type_ascription;
                     self.cx.current_expansion.prior_type_ascription = mac.prior_type_ascription;
-                    let tok_result = expander.expand(self.cx, span, mac.args.inner_tokens());
+                    // FIXME - this is imprecise. We should try to lint as close to the macro call
+                    // as possible (we cannot take attributes from the macro call itself).
```

Comment:
If you want to add a FIXME, it's better to add it to `fn lint_node_id` itself, since these words apply to every its use.